libHttpClient.props

```diff
@@ -33,6 +34,7 @@
     <useWsppWebsockets>false</useWsppWebsockets>
     <useWsppWebsockets Condition="'$(HCLibPlatformType)'=='Win32'">true</useWsppWebsockets>
 
+    <HCLibToolset>$(PlatformToolsetVersion)</HCLibToolset>
     <HCLibToolset Condition="'$(PlatformToolset)'=='v140'">140</HCLibToolset>
```

Comment:
redundant since v140 support is dropped?

Reply:
ya should be removed